Src/VimCore/CommonOperations.fs

```diff
@@ -1123,16 +1123,18 @@ type internal CommonOperations
             Count = VimRegexReplaceCount.One }
 
     member x.GoToDefinition() =
-        let before = TextViewUtil.GetCaretVirtualPoint _textView
-        if _vimHost.GoToDefinition() then
-            _jumpList.Add before
-            Result.Succeeded
-        else
-            match _wordUtil.GetFullWordSpan WordKind.BigWord _textView.Caret.Position.BufferPosition with
-            | Some(span) -> 
-                let msg = Resources.Common_GotoDefFailed (span.GetText())
-                Result.Failed(msg)
-            | None ->  Result.Failed(Resources.Common_GotoDefNoWordUnderCursor) 
+        match x.WordUnderCursorOrEmpty with
+        | "" ->
+            Result.Failed(Resources.Common_GotoDefNoWordUnderCursor) 
+        | word when word.StartsWith("http") ->

```

Comment:
This will be a case sensitive comparison.

Reply:
Who are these people who use upper case for URLs! :smile:

Reply:
I also noticed that will accidentally attempt to open a link when a big-word delimited identifier starts with, say, `Http`, which is actually pretty likely in a lot of networking code. I changed it to case-insensitive prefix of `http:` or `https:`.